examples/src/test/java/io/coti/sdk/TokenGenerationTest.java

```diff
@@ -0,0 +1,74 @@
+package io.coti.sdk;
+
+import io.coti.basenode.crypto.CryptoHelper;
+import io.coti.basenode.crypto.OriginatorCurrencyCrypto;
+import io.coti.basenode.data.*;
+import io.coti.basenode.http.GenerateTokenFeeRequest;
+import io.coti.sdk.http.AddTransactionRequest;
+import io.coti.sdk.http.FullNodeFeeResponse;
+import io.coti.sdk.utils.CryptoUtils;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static io.coti.sdk.utils.Constants.NATIVE_CURRENCY_SYMBOL;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class TokenGenerationTest {
```

Comment:
if decided that the name of the class should be without Example like in TransferExampleTest then please rename that file